CMSgov/cms-carts-seds
# Dependencies/06 2022

# Description
Whole host of dependency updates across UI-Src. Updates some easy major dependencies, removes unused dependencies, and migrates old ones that are no longer supported (Node-sass). Doesn't do everything, but updates quite a bit

## How to test

./dev local

Try just about everything in the app to make sure nothing funky happens

## Dependencies

Miiiight wanna look at package.json for this one ;)

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
